### PR TITLE
Change trunk eni table id column to big integer.

### PR DESCRIPTION
**Pull request by aloktiagi**
Change trunk eni table id column to big integer.

### CODE REVIEW

File: vpc/service/db/migrations/38_trunk_eni_id.down.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+START TRANSACTION;
+LOCK TABLE htb_classid IN ACCESS EXCLUSIVE MODE;
+LOCK TABLE trunk_enis IN ACCESS EXCLUSIVE MODE;
+ALTER SEQUENCE trunk_enis_id_seq AS INTEGER NO CYCLE;
```
Comment (MyUmmaGumma) [inline vpc/service/db/migrations/38_trunk_eni_id.down.sql:4]: Does it matter that this is back to `int` even though during the incident resolution it was marked as `bigint`?

Comment (sargun) [reply vpc/service/db/migrations/38_trunk_eni_id.down.sql:4]: it'll be a problem if we've exceeeded 2**31.

Comment (aloktiagi) [reply vpc/service/db/migrations/38_trunk_eni_id.down.sql:4]: @MyUmmaGumma thats the `down` migration file to migrate down to a previous version.

Comment (MyUmmaGumma) [reply vpc/service/db/migrations/38_trunk_eni_id.down.sql:4]: Understood, I was wondering if the previous version is one on the schema or the one on the box(which is bigint now)

**Review by MyUmmaGumma — APPROVED** [review]

**Pull request merged**